parachain-template/node/src/chain_spec.rs

```diff
@@ -172,7 +172,7 @@ pub fn local_testnet_config() -> ChainSpec {
 		// Extensions
 		Extensions {
 			relay_chain: "rococo-local".into(), // You MUST set this to the correct network!
-			para_id: 1000,
+			para_id: 1001,
```

Comment:
Can we override the parachain in the zombienet launch config?

Reply:
Yes, would prefer explict it not conflicts with `asset_hub`.